src/main/java/uk/gov/hmcts/reform/sendletter/ResponseExceptionHandler.java

```diff
@@ -49,24 +50,29 @@ protected void activateDirectFieldAccess(DataBinder dataBinder) {
     }
 
     @ExceptionHandler(InvalidTokenException.class)
-    protected ResponseEntity handleInvalidTokenException() {
+    protected ResponseEntity<Void> handleInvalidTokenException() {
```

Comment:
Is there any benefit of specifying generic parameters?

Reply:
solves unchecked type warning - should avoid that